src/julia-parser.scm

```diff
@@ -15,7 +15,7 @@
 (define prec-conditional '(?))
 (define prec-arrow       (append!
                           '(-- -->)
-                          (add-dots '(← → ↔ ↚ ↛ ↞ ↠ ↢ ↣ ↦ ↤ ↮ ⇎ ⇍ ⇏ ⇐ ⇒ ⇔ ⇴ ⇶ ⇷ ⇸ ⇹ ⇺ ⇻ ⇼ ⇽ ⇾ ⇿ ⟵ ⟶ ⟷ ⟹ ⟺ ⟻ ⟼ ⟽ ⟾ ⟿ ⤀ ⤁ ⤂ ⤃ ⤄ ⤅ ⤆ ⤇ ⤌ ⤍ ⤎ ⤏ ⤐ ⤑ ⤔ ⤕ ⤖ ⤗ ⤘ ⤝ ⤞ ⤟ ⤠ ⥄ ⥅ ⥆ ⥇ ⥈ ⥊ ⥋ ⥎ ⥐ ⥒ ⥓ ⥖ ⥗ ⥚ ⥛ ⥞ ⥟ ⥢ ⥤ ⥦ ⥧ ⥨ ⥩ ⥪ ⥫ ⥬ ⥭ ⥰ ⧴ ⬱ ⬰ ⬲ ⬳ ⬴ ⬵ ⬶ ⬷ ⬸ ⬹ ⬺ ⬻ ⬼ ⬽ ⬾ ⬿ ⭀ ⭁ ⭂ ⭃ ⭄ ⭇ ⭈ ⭉ ⭊ ⭋ ⭌ ￩ ￫ ⇜ ⇝ ↜ ↝ ↩ ↪ ↫ ↬ ↼ ↽ ⇀ ⇁ ⇄ ⇆ ⇇ ⇉ ⇋ ⇌ ⇚ ⇛ ⇠ ⇢ ↷ ↶ ↺ ↻))))
+                          (add-dots '(← → ↔ ↚ ↛ ↞ ↠ ↢ ↣ ↦ ↤ ↮ ⇎ ⇍ ⇏ ⇐ ⇒ ⇔ ⇴ ⇶ ⇷ ⇸ ⇹ ⇺ ⇻ ⇼ ⇽ ⇾ ⇿ ⟵ ⟶ ⟷ ⟹ ⟺ ⟻ ⟼ ⟽ ⟾ ⟿ ⤀ ⤁ ⤂ ⤃ ⤄ ⤅ ⤆ ⤇ ⤌ ⤍ ⤎ ⤏ ⤐ ⤑ ⤔ ⤕ ⤖ ⤗ ⤘ ⤝ ⤞ ⤟ ⤠ ⥄ ⥅ ⥆ ⥇ ⥈ ⥊ ⥋ ⥎ ⥐ ⥒ ⥓ ⥖ ⥗ ⥚ ⥛ ⥞ ⥟ ⥢ ⥤ ⥦ ⥧ ⥨ ⥩ ⥪ ⥫ ⥬ ⥭ ⥰ ⧴ ⬱ ⬰ ⬲ ⬳ ⬴ ⬵ ⬶ ⬷ ⬸ ⬹ ⬺ ⬻ ⬼ ⬽ ⬾ ⬿ ⭀ ⭁ ⭂ ⭃ ⭄ ⭇ ⭈ ⭉ ⭊ ⭋ ⭌ ￩ ￫ ⇜ ⇝ ↜ ↝ ↩ ↪ ↫ ↬ ↼ ↽ ⇀ ⇁ ⇄ ⇆ ⇇ ⇉ ⇋ ⇌ ⇚ ⇛ ⇠ ⇢ ↷ ↶ ↺ ↻ <:))))
```

Comment:
Also `:>` ?

Reply:
@stevengj 
I'm not sure? Seems that `:>` is seen by julia as either as a  `>` operator or a Symbol?
```
julia> Int <: Int
true

Int :> Int
ERROR: syntax: ">" is not a unary operator

julia> Symbol(<:)
:<:

julia> Symbol(:>)
:>

help?> <:
search: <:

  <:(T1, T2)

help?> :>
  Symbol
  The type of object used to represent identifiers in parsed julia code (ASTs). Also often used as a name or label to identify an entity (e.g. as a dictionary key). Symbols can be entered using the :
  quote operator:

```
Adding the change results in error during the `make`:
```
JULIA usr/lib/julia/corecompiler.ji
error during bootstrap:
LoadError(at "compiler/compiler.jl" line 3: LoadError(at "compiler/abstractinterpretation.jl" line 224: ErrorException("syntax: unexpected ")"")))
jl_f_throw at /Users/ssikdar1/julia/src/builtins.c:442
jl_apply at /Users/ssikdar1/julia/src/./julia.h:1696 [inlined]
do_call at /Users/ssikdar1/julia/src/interpreter.c:369
eval_stmt_value at /Users/ssikdar1/julia/src/interpreter.c:409 [inlined]
eval_body at /Users/ssikdar1/julia/src/interpreter.c:807
jl_interpret_toplevel_thunk at /Users/ssikdar1/julia/src/interpreter.c:915
top-level scope at compiler/abstractinterpretation.jl:224
jl_toplevel_eval_flex at /Users/ssikdar1/julia/src/toplevel.c:813
jl_eval_errorf at /Users/ssikdar1/julia/src/toplevel.c:583
jl_toplevel_eval_flex at /Users/ssikdar1/julia/src/toplevel.c:772
jl_parse_eval_all at /Users/ssikdar1/julia/src/ast.c:907
jl_load_rewrite at /Users/ssikdar1/julia/src/toplevel.c:871 [inlined]
jl_load at /Users/ssikdar1/julia/src/toplevel.c:876 [inlined]
jl_load_ at /Users/ssikdar1/julia/src/toplevel.c:883
unknown function (ip: 0x106088e62)
unknown function (ip: 0x106088d57)
jl_apply at /Users/ssikdar1/julia/src/./julia.h:1696 [inlined]
do_call at /Users/ssikdar1/julia/src/interpreter.c:369
eval_stmt_value at /Users/ssikdar1/julia/src/interpreter.c:409 [inlined]
eval_body at /Users/ssikdar1/julia/src/interpreter.c:807
jl_interpret_toplevel_thunk at /Users/ssikdar1/julia/src/interpreter.c:915
top-level scope at compiler/compiler.jl:109
jl_toplevel_eval_flex at /Users/ssikdar1/julia/src/toplevel.c:813
jl_eval_module_expr at /Users/ssikdar1/julia/src/toplevel.c:180
jl_toplevel_eval_flex at /Users/ssikdar1/julia/src/toplevel.c:639
jl_toplevel_eval at /Users/ssikdar1/julia/src/toplevel.c:822 [inlined]
jl_toplevel_eval_in at /Users/ssikdar1/julia/src/toplevel.c:842
unknown function (ip: 0x106087df2)
jl_apply at /Users/ssikdar1/julia/src/./julia.h:1696 [inlined]
do_call at /Users/ssikdar1/julia/src/interpreter.c:369
eval_stmt_value at /Users/ssikdar1/julia/src/interpreter.c:409 [inlined]
eval_body at /Users/ssikdar1/julia/src/interpreter.c:807
jl_interpret_toplevel_thunk at /Users/ssikdar1/julia/src/interpreter.c:915
top-level scope at compiler/compiler.jl:3
jl_toplevel_eval_flex at /Users/ssikdar1/julia/src/toplevel.c:813
jl_parse_eval_all at /Users/ssikdar1/julia/src/ast.c:907
jl_load_rewrite at /Users/ssikdar1/julia/src/toplevel.c:871 [inlined]
jl_load at /Users/ssikdar1/julia/src/toplevel.c:876
exec_program at /Users/ssikdar1/julia/usr/bin/julia (unknown line)
true_main at /Users/ssikdar1/julia/usr/bin/julia (unknown line)
main at /Users/ssikdar1/julia/usr/bin/julia (unknown line)
```

Reply:
I think Steven meant `>:`, the supertype operator.

Reply:
oh that makes more sense!

Reply:
added `.>:` to `add-dots` in 2fca8fa